package.json

```diff
@@ -39,7 +39,7 @@
     }
   },
   "scripts": {
-    "dev": "vite",
```

Comment:
I don't see an index.hml within the repo and whe'person I tried it just serves a blank page, am I missing something?

Reply:
We do have it set up so that you could add a top level `dev` folder with an index.html and it should serve it. It was in fact part of the repo until recently but was added to `gitignore` not to pollute the repo
https://github.com/inocan-group/vue3-google-map/tree/v0.17.0/dev